### DISCUSSION TITLE
How to use one of the columns of the function percentile to display a leaderboard on web ui.

### DISCUSSION THREAD

**Discussion by csh995426531** [apache/skywalking#10047] [Q&A] [answered]
What I want to expect is to be able to display a list of endpoints sorted by p95 or p99 on the official web ui. Maybe it's because of multiple value that I can't use `get sorted top N values`, I just got into this project, and I don't know much about the logic of OAL syntax, so I'd like to ask what I need to do to achieve my goal, and I'd like to get some help.

**Comment by wu-sheng** [comment, marked as answer]
No, you can't. Because `percentile` is providing more efficiency by using labeled metrics. 

You could extend and write p95 and p99 metrics on your own, as a single value metric. Then you could have a top.

From SkyWalking perspective, ordering by p99 doesn't make sense in real use case.

**Reply by csh995426531** [reply]
Thankfully, I got the answer I wanted and I will reconsider whether this makes sense.
